src/test/resources/db/orgDataSwapBiodata.sql

```diff
@@ -0,0 +1 @@
+select create_swap_table ('biodata', 'wqp', 'org_data');
```

Comment:
Not used? Should be (and is) part of test_db.sql

Reply:
I see - deleted!